x-pack/plugins/apm/public/components/app/ServiceDetails/ServiceDetailTabs.tsx

```diff
@@ -29,7 +30,19 @@ interface Props {
 
 export function ServiceDetailTabs({ serviceName, tab }: Props) {
   const { agentName } = useAgentName();
-  const { serviceMapEnabled } = useApmPluginContext().config;
+  const { uiSettings } = useApmPluginContext().core;
+
+  const overviewTab = {
+    link: (
+      <a title="UNDER CONSTRUCTION" href="#">
```

Comment:
is `UNDER CONSTRUCTION` intentional?

Reply:
Yes. It's just a placeholder and behind the flag and will be replaced in #81718.